### ISSUE TITLE
Left Stick Down Rigth direction not detected properly

### ISSUE THREAD

**Issue by jotacee98** [MTCKC/ProconXInput#57]
Hi, after calibrating the controller the left stick doesnt fully takes the down right direction, I mean if Im playing a third person game the character doesnt run backwards to the right, he just walks, it doesnt take the full range of the stick. Has this happened to anyone else? Every other direction works well.

**Comment by jarrey68** [comment]
YES this is happening to me right now. it is not a problem on switch for some reason.
